### ISSUE TITLE
GetLoadedDeviceOfType doesn't return a vector of strings as comment suggests

### ISSUE THREAD

**Issue by HarmlessHarm** [nicost/micro-manager#88]
https://github.com/nicost/micro-manager/blob/81eea2d445f755beb6d1c384f40eb2155c6ab5cf/MMDevice/DeviceBase.h#L973-L975

This function doesn't return a vector<string> of loaded devices of type as the comment suggests, but requires an index to return the *char of the device at given index.

**Comment by nicost** [comment]
Thanks!  I pushed a better comment into the svn repo.  It looks like the function was changed and the comment was not.  B.t.w., it is not clear to me how to get the number of loaded devices of the given type (which seems to be needed to use this function without resorting to awful approaches).  Let me know if that is indeed so, and I can add that function (GetNumberOfLoadedDevicesOfType or something like that).
